waiter/src/waiter/token.clj

```diff
@@ -668,6 +690,12 @@
                           (string? owner-param) #{owner-param}
                           (coll? owner-param) (set owner-param)
                           :else (list-token-owners kv-store))
+                 include-param (get request-params "include")
```

Comment:
It seems that the `include` query parameter will have several meanings. It is used for `metadata`, `deleted`, `service-id`, `cluster`; maybe we can create a new query field name instead? Does `fields` sound okay? so:
`/tokens?fields=metadata` will provide token `{"metadata": {}, "token":"kitchen", ...other default index fields`, and still includes the default fields.

Here is an example `fields` interface: https://developers.facebook.com/docs/graph-api/overview#fields

Reply:
The response will not have a nested `parameters` response anymore

Reply:
We spoke offline and decided to use the `parameters` query parameter. We did not go with a flat map due to the conflict with the `maintenance` field already displayed in the `/tokens` output today.